### PR TITLE
+ language to loan-basics/benefits-and-limitations

### PR DESCRIPTION
**Pull request by JamelleWT**
This language, formerly referred to as "loan basics" is actually loan-basics/benefits and limitations. I have pasted the markdown-formatted content. please review

### CODE REVIEW

File: pages/loan-basics/benefits-and-limitations.md [hunk 1]
```diff
@@ -12,3 +12,100 @@ scripts:
...
+
+## Loan types and terms
+We allow two types of loans:
+
+#### General purpose
++ May be used for any purpose
++ Requires no documentation
++ Has a repayment term of 1 to 5 years
+
+#### Residential
```
Comment (line47) [inline pages/loan-basics/benefits-and-limitations.md]: I think these 2 headers should be `###` for General purpose and Loan types and terms

File: pages/loan-basics/benefits-and-limitations.md [hunk 2]
```diff
@@ -12,3 +12,100 @@ scripts:
...
+**To be eligible for a loan, you**
++ must be an active federal employee or a member of the uniformed services.
++ must be in pay status because repayments are set up as payroll deductions.
++ can only have one outstanding general purpose loan and one outstanding residential loan from any one TSP account at a time.
++ must have at least $1,000 of your own contributions and earnings in your account (agency contributions and earnings cannot be borrowed).
++ must not have repaid a loan of the same type in full within the past 60 days. (If you have both a civilian account and a uniformed services account, the 60-day waiting period applies separately to each account.)
++ must not have had a taxable distribution of a loan within the past 12 months unless it was the result of your separation from federal service.
++ must not have a court order against your account.
+
+#### Residential loan eligibility rules
```
Comment (line47) [inline pages/loan-basics/benefits-and-limitations.md]: I think these 2 headers should be `###` for General eligibility rules and Residential loan eligibility rules

File: pages/loan-basics/benefits-and-limitations.md [hunk 3]
```diff
@@ -12,3 +12,100 @@ scripts:
...
+. . + renovations to your existing residence;
+. . + buying out another person's share in the your current residence; or
+. . + the purchase of land only.
+ 
++ Your primary residence must be purchased in whole or in part by you, or your spouse.
+
+## Borrowing limits
+#### Minimum loan amount
+The minimum loan amount that you can borrow is $1,000. The amount of your account balance that consists of your own contributions and earnings on those contributions, must be at least equal to the minimum loan amount. 
+#### Maximum loan amount
```
Comment (line47) [inline pages/loan-basics/benefits-and-limitations.md]: I think these 2 headers should be `###` for Minimum loan amount and Maximum loan amount

File: pages/loan-basics/benefits-and-limitations.md [hunk 4]
```diff
@@ -12,3 +12,100 @@ scripts:
...
+
+## Loan costs
+
+#### Direct costs
+
+_**Loan Fee**_ We charge a loan fee of $50 for administrative expenses. This fee is deducted from your loan proceeds. For example, if you request a loan for $1,000, the amount paid to you will be $950.
+ 
+_**Interest**_ The interest rate on your loan is the G Fund rate at the time your loan application is processed. This rate is fixed for the life of the loan. Although loan interest is not tax-deductible, all of the interest goes back into your TSP account.
+#### Indirect costs
+When you take a loan, you sacrifice the earnings that might have accrued on the borrowed money, had it remained in your TSP account.
```
Comment (line47) [inline pages/loan-basics/benefits-and-limitations.md]: I think these 2 headers should be `###` for Direct costs and Indirect costs

File: pages/loan-basics/benefits-and-limitations.md (hunk 1 above)
Comment (KishaT) [reply pages/loan-basics/benefits-and-limitations.md]: I hit resolve on one of your comments to see what would happen. :) I think you're right, the headers with four ####, should be three, ###. Please make the changes as necessary.

**Pull request merged**